index.js

```diff
@@ -44,12 +44,18 @@ const getSchemaProperties = (props) => {
     const original = props[key]
 
     // Skip props that have '@ignore' in description (eg. in material-ui)
-    if (original.description.indexOf('@ignore') > -1) {
-      return result
-    }
+    if (typeof original.description === 'undefined' && typeof original.type !== 'undefined') {
```

Comment:
When using PropTypes.shape with a nested object, the nested properties come through this function with description hanging off of `original.type` and not `original`